src/main/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersController.java

```diff
@@ -40,39 +42,98 @@
+    public Page<CasePaymentOrderEntity> getCasePaymentOrders(@ApiParam("list of IDs")
                                                              @ValidCpoId
-                                                             @RequestParam("ids") Optional<List<String>> ids,
-                                                             @ApiParam("casesId of ids")
+                                                             @RequestParam(name = IDS, required = false)
+                                                                         Optional<List<String>> ids,
+                                                             @ApiParam("list of caseIDs")
                                                              @ValidCaseId
-                                                             @RequestParam("cases-ids") Optional<List<String>> casesId,
-                                                             @RequestParam("pageSize") Optional<Integer> pageSize,
-                                                             @RequestParam("pageNumber") Optional<Integer> pageNumber
+                                                             @RequestParam(name = CASE_IDS, required = false)
+                                                                     Optional<List<String>> casesId,
+                                                             @RequestParam(name = "pageSize", required = false)
+                                                                         Optional<Integer> pageSize,
+                                                             @RequestParam(name = "pageNumber", required = false)
+                                                                         Optional<Integer> pageNumber
 
     ) {
-        final List<String> listOfIds = ids.orElse(Collections.emptyList());
-        final List<String> listOfCasesIds = casesId.orElse(Collections.emptyList());
+        final List<String> listOfIds = ids.orElse(emptyList());
+        final List<String> listOfCasesIds = casesId.orElse(emptyList());
         final CasePaymentOrderQueryFilter casePaymentOrderQueryFilter = CasePaymentOrderQueryFilter.builder()
-            .listOfIds(listOfIds)
-            .listOfCasesIds(listOfCasesIds)
-            .pageNumber(pageNumber.orElse(Integer.parseInt(applicationParams.getDefaultPageNumber())))
-            .pageSize(pageSize.orElse(Integer.parseInt(applicationParams.getDefaultPageSize())))
-            .build();
+                .listOfIds(listOfIds)
+                .listOfCasesIds(listOfCasesIds)
+                .pageNumber(pageNumber.orElse(Integer.parseInt(applicationParams.getDefaultPageNumber())))
+                .pageSize(pageSize.orElse(Integer.parseInt(applicationParams.getDefaultPageSize())))
+                .build();
         return casePaymentOrdersService.getCasePaymentOrders(casePaymentOrderQueryFilter);
     }
 
+    @DeleteMapping(path = CASE_PAYMENT_ORDERS_PATH)
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Minor, but since we're not returning a body
```suggestion
    @ResponseStatus(HttpStatus.NO_CONTENT)
```